.github/workflows/ci.yml

```diff
@@ -66,6 +66,37 @@ jobs:
           node-version: ${{ matrix.node }}
       - run: corepack yarn
       - run: corepack yarn vitest run --coverage ./test/unit
+      - name: Upload coverage reports
+        if: matrix.node == 22 # Only upload coverage from the latest Node.js version
+        uses: actions/upload-artifact@v4
+        with:
+          name: coverage-reports
+          path: coverage/
+      - name: Upload coverage to Codecov
```

Comment:
Personally I’m not a fan of using the `name` property too much. It acts as a label, which defaults to the action name. IMO `codecov/codecov-action@v4` is more descriptive than a custom label.

Reply:
I agree that name is not needed when the `uses` is descriptive, however in this case I don't think `codecov/codecov-action@v4` is more descriptive than `Upload coverage to Codecov`. it's not obvious that `codecov/codecov-action@v4` does just an upload.